game.py

```diff
@@ -130,6 +140,8 @@ def _all_clear(self):
             self._correct = 0
 
         self._set_label('')
+        for q in self._question:
```

Comment:
Would be nice to use a better variable name for q, like question. Would help for readability.

Reply:
Sure, I'll change it.